CircleciScripts/upload_xcframeworks.py

```diff
@@ -0,0 +1,25 @@
+import boto3
+from botocore.exceptions import ClientError
+from framework_list import xcframeworks
+from functions import log
+
+def upload_file(file_name, bucket, object_name): 
```

Comment:
`object_name` will include the "prefix" or "directory", right? Are we going to upload directly to the bucket root, or to different prefixes for SDK and AppSync? Either is fine, just making sure you've considered.

Reply:
Added `iosv2sdk`